src/components/MessageLogger/LineMessage.tsx

```diff
@@ -1,15 +1,22 @@
 import React, { useEffect, useRef, useState } from "react";
 import styles from "@components/MessageLogger/LineMessage.module.scss";
-import { IoIosArrowDropdownCircle } from "react-icons/io";
 import { BsFillCaretRightFill } from "react-icons/bs";
 import { Message } from "@adapters/Message";
+import { Counter } from "@components/MessageLogger/Counter";
 
 interface Props {
   message: Message;
   count: number;
+  color: string;
+  colorBackground: string;
```

Comment:
The background color shouldnt be passed as a function, but calculated from the color parameter. Look into HSL Colors.